cabot/cabotapp/models.py

```diff
@@ -542,7 +536,8 @@ def description(self):
             return 'ICMP Reply from {}'.format(self.instance_set.all()[0].address)
         return 'ICMP Reply'
 
-    update_url = 'update-icmp-check'
+    # Unless we unlink this status from everywhere, we just need to provide an alternative url.
+    update_url = 'update-jenkins-check'
```

Comment:
Regardless, this entirely block of code will be removed in the TCP commit, so should I even bother with any changes. And on that note, should we wait to do extensive QA until a everything is in?